sympy/physics/control/lti.py

```diff
@@ -20,8 +20,42 @@ def _roots(poly, var):
         r = [rootof(poly, var, k) for k in range(n)]
     return r
 
+class LTICommon:
+    """A common class for all the Linear Time-Invariant Dynamical Systems."""
```

Comment:
I wonder if a better name is possible than `LTICommon`. Ideally the name would be usable in the docs to describe what is allowed as e.g. input to a particular function.

Reply:
This docstring could describe what properties or methods can be expected to be provided by any instance of LTICommon